apis/v1alpha1/httproute_types.go

```diff
@@ -22,6 +22,7 @@ import (
 // +genclient
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Hostnames",type=string,JSONPath=`.spec.hostnames`
```

Comment:
What does this output look like if there are multiple hostnames specified?

Reply:
@robscott it looks like this:

```
kubectl get httproute
NAME                  HOSTNAMES
default-match-route   ["default-match.com"]
http-app-1            ["baz.example.com","baz2.example.com","baz3.example.com"]
http-trafficsplit-1   ["my.trafficsplit.com"]
```